frame/nomination-pools/benchmarking/src/mock.rs

```diff
@@ -112,13 +112,20 @@ impl pallet_staking::Config for Runtime {
 	type ElectionProvider =
 		frame_election_provider_support::NoElection<(AccountId, BlockNumber, Staking, ())>;
 	type GenesisElectionProvider = Self::ElectionProvider;
-	type VoterList = VoterList;
+	type VoterList = <Runtime as pallet_stake_tracker::Config>::VoterList;
 	type TargetList = pallet_staking::UseValidatorsMap<Self>;
 	type MaxUnlockingChunks = ConstU32<32>;
 	type HistoryDepth = ConstU32<84>;
 	type OnStakerSlash = Pools;
 	type BenchmarkingConfig = pallet_staking::TestBenchmarkingConfig;
 	type WeightInfo = ();
+	type EventListeners = StakeTracker;
+}
+
+impl pallet_stake_tracker::Config for Runtime {
```

Comment:
Yes, we can use VoterList directly here. StakeTracker is needed, because without it benchmark tests fail, it has been introduced for a reason. 
Basically we can use VoterList directly everywhere, but it seems like a good idea to actually wrap it with `TrackedList` abstraction for good usage examples, that way the users are going to have a better understanding. It would be even better if we could restrict staking to that type for both `VoterList` and `TargetList`, but then it needs to be moved to `primitives`.